utils/build-revive.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+REVIVE_INSTALL_DIR=$(pwd)/target/release
+while getopts "o:" option ; do
+    case $option in
+    o) # Output directory
+        REVIVE_INSTALL_DIR=$OPTARG
+        ;;
+    \?) echo "Error: Invalid option"
+        exit;;
```

Comment:
Should probably return non-zero value here?

```suggestion
        exit 1;;
```

Reply:
Done